manifests/base/kustomization.yaml

```diff
@@ -16,3 +9,10 @@ images:
 - name: argoproj/argocd-ui
   newName: argoproj/argocd-ui
   newTag: latest
+resources:
```

Comment:
All of the `manifests/*` changes are due to using the new version (v2.1.0) of kustomize to generate them.